.github/workflows/main.yml

```diff
@@ -1,24 +1,24 @@
-name: Tests
+name: Memories
 
-on: pull_request
+on: push
 
 jobs:
   build:
-    runs-on: ubuntu-latest
+    runs-on: macOS-latest
     steps:
-    - name: build
-      run: echo hello world
+    - uses: actions/checkout@v2
+    - run: xcodebuild -quiet -project Memories.xcodeproj -scheme Memories | xcpretty
```

Comment:
This command (and the similar commands for unit and UI tests) fail because I don't have have a development team setup. Yet each job passes. Latest action: https://github.com/austinrovge/memories/runs/5076456493?check_suite_focus=true

Why is Github actions so weird to use?

Reply:
Piping into `xcpretty` was ignoring the exit code of `xcodebuild`. Not a fault of Github actions